source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```
@@ -72,46 +99,51 @@ a {
-  top: 2rem;
-  display: block;
-  overflow-x: hidden;
-  padding-left: 4.5rem;
-  align-items: stretch;
-  width: 100%;
-}
-.mainArea.menuShowing {
-  height: var(--mainAreaHeight);
+    right: 6rem;
+    /* top: 2rem; */
+    display: block;
```

Stylelint (17.7.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)